lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1501,6 +1510,7 @@ class SemanticsObject {
       PrimaryRole.image => ImageRoleManager(this),
       PrimaryRole.platformView => PlatformViewRoleManager(this),
       PrimaryRole.generic => GenericRole(this),
+      PrimaryRole.link => Link(this),
```

Comment:
Let's keep the `generic` role last to underscore that it's the fallback role that we only use as a last resort.